tsconfig.common.json

```diff
@@ -21,6 +18,7 @@
     "noUnusedParameters": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "strict": true
+    "strict": true,
+    "types": ["ava"] // Add this line
```

Comment:
This line doesn't feel significant. This just gives the typescript compiler a hint as to where ava type definitions are.

I don't think fixes our problem - we need to tell ava how to load typescript files